webapp/src/modules/auction/reducer.js

```diff
@@ -81,18 +85,18 @@ export function auctionReducer(state = INITIAL_STATE, action) {
     }
     case FETCH_AUCTION_PARAMS_SUCCESS: {
       const { params } = action
-
       return {
         loading: loadingReducer(state.loading, action),
         error: null,
         data: {
           ...state.data,
-          params: {
-            availableParcelCount: params.availableParcelCount,
-            gasPriceLimit: params.gasPriceLimit,
-            landsLimitPerBid: params.landsLimitPerBid,
-            currentPrice: params.currentPrice
-          }
+          params: Object.keys(params).reduce(
+            (acc, key) => ({
+              ...acc,
+              [key]: params[key]
+            }),
+            {}
+          )
```

Comment:
This is 👍 , but I wanted to explain. The reasoning behind not using a reduce here (in mind) was not setting incorrect params by accident.
Maybe we can use 

```javascript
utils.pick(params, Object.keys(INITIAL_STATE.data.params))
```

what do you think?